### ISSUE TITLE
Render Collection with index

### ISSUE THREAD

**Issue by roman-dubrovsky** [datarockets/rails-views#11]
= concept("welcome/top_participant/cell", collection: top5_users_game_sessions) 

Also need to add the element index in the collection

**Comment by DDKatch** [comment]
@roman-dubrovsky now we can go with smth like this:
**TableForSmthCell** view part:
```
- collection.each_with_index do |element, index|
  = concept('welcome/top_participant/cell', element: {element, index})
- end
```
and we can archive the same functionality.

**Comment by roman-dubrovsky** [comment]
@DDKatch heh, I don't remember anything about this issue 😕

**Comment by AleksSenkou** [comment]
Sounds reasonable thing to have for me.

We can go with:

```ruby
= concept("welcome/top_participant/cell", collection: top5_users_game_sessions, index: true)
```

And mark `index` as false by default


Thoughts?

**Comment by DDKatch** [comment]
@AleksSenkou I thought that according to namespace 'welcome/top_participant/cell' it should represent single model for each element from collection. But even if it's for collection, what view we should use for a single one?

**Comment by AleksSenkou** [comment]
@DDKatch yeah, it represents single model for each element from collection

**Comment by AleksSenkou** [comment]
@DDKatch please, leave here a summary of yesterday's discussion

**Comment by DDKatch** [comment]
@AleksSenkou I got your point that it's more convenient to use separate logic for that.
